### PR TITLE
replace tcga studies with msk impact

### PR DESCRIPTION
**Pull request by JiaoJiao123**

### CODE REVIEW

File: app/views/gene.html [hunk 1]
```diff
@@ -115,41 +115,16 @@
...
-                <li><img src="resources/images/nci.png"/> <span><a target="_blank" href="http://www.cbioportal.org/study.do?cancer_study_id=stad_tcga_pub">Stomach Adenocarcinoma (TCGA, Nature 2014)</a></li>
-                <li><img src="resources/images/nci.png"/> <span><a target="_blank" href="http://www.cbioportal.org/study.do?cancer_study_id=tgct_tcga">Testicular Germ Cell Cancer (TCGA, Provisional)</a></li>
-                <li><img src="resources/images/nci.png"/> <span><a target="_blank" href="http://www.cbioportal.org/study.do?cancer_study_id=ucec_tcga_pub">Uterine Corpus Endometrioid Carcinoma (TCGA, Nature 2013)</a></li>
-             </ul>'></i></span></span></div>
+            <div class="mutation-mapper-title"><span>Annotated Mutation Distribution in <a ng-if="studyName.indexOf('(') !== -1" href="http://www.cbioportal.org/study?id=msk_impact_2017#summary" target="_blank">{{studyName}}</a>
+                <span ng-if="studyName.indexOf('(') === -1">{{studyName}}</span></span></div>
```
Comment (zhx828) [inline app/views/gene.html]: Why we have this check?

File: app/scripts/controllers/gene.js [hunk 2]
```diff
@@ -502,15 +498,14 @@ angular.module('oncokbStaticApp')
                 }
             });
         }
-
+        $scope.studyName = 'MSK-IMPACT Clinical Sequencing Cohort (MSKCC, Nat Med 2017)';
```
Comment (zhx828) [inline app/scripts/controllers/gene.js]: `studyName` has been assigned in multiple places. Are they necessary? If so, could you make this description a constant and to be used in different places?

File: app/scripts/controllers/gene.js [hunk 3]
```diff
@@ -451,7 +447,7 @@ angular.module('oncokbStaticApp')
                 },
                 xaxis: {
                     tickfont: {
-                        size: Math.max(Math.min(260 / maxLengthStudy.length, 15, 180 / shortNames.length), 6)
+                        size: Math.max(Math.min(260 / maxLengthStudy.length, 15, 180 / shortNames.length), 10)
```
Comment (zhx828) [inline app/scripts/controllers/gene.js]: if `maxLengthStudy.length ` or `shortNames.length` is 0, what's gonna happen?

File: app/scripts/controllers/gene.js [hunk 4]
```diff
@@ -666,7 +665,13 @@ angular.module('oncokbStaticApp')
             inGenePage: false,
             inVariantPage: false
         };
-
+        if ($routeParams.geneName) {
```
Comment (zhx828) [inline app/scripts/controllers/gene.js]: Why moving the code would solve the issue?

**Comment by JiaoJiao123** [comment]
Solved Issue #153

**Pull request merged**